images/gas-sponsor-emulator/server.py

```
@@ -0,0 +1,272 @@
+        if receipt is not None:
+            return receipt
+        rpc("anvil_mine", ["0x1"])
+        time.sleep(0.5)
+    raise RuntimeError(f"timed out waiting for receipt {tx_hash}")
+
+
+def verify_signature(address: str, raw: str, signature: str, no_hash=False):
+    command = ["cast", "wallet", "verify", "--address", address]
+    if no_hash:
+        command.append("--no-hash")
+    subprocess.check_call(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)